### PR TITLE
create card

### PR DESCRIPTION
**Pull request by OksanaGrygorchuk**
https://OksanaGrygorchuk.github.io/layout_product-cards/
https://OksanaGrygorchuk.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,47 @@
...
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```
Comment (nazarmatsevych) [inline src/index.html:6]:
```suggestion
    <meta
      name="viewport"
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,47 @@
...
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="iMac photo"
+      >
+      <p class="card__discription">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+
+      <div class="card__code">Product code: 195434</div>
```
Comment (nazarmatsevych) [inline src/index.html:22]: Use semantic tags for text content like `<p>` or `<h2>`...

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,47 @@
...
+      <div class="card__price-container">
+        <span class="card__price-label">Price</span>
+        <span class="card__price">$2,199</span>
+      </div>
+
+      <a
+        class="card__buy"
+        href="#"
+        data-qa="hover"
+      >buy</a>
```
Comment (nazarmatsevych) [inline src/index.html:44]:
```suggestion
      <a
        class="card__buy"
        href="#"
        data-qa="hover"
      >
        buy
      </a>
```

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,47 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="iMac photo"
```
Comment (nazarmatsevych) [inline src/index.html:18]: image

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (OksanaGrygorchuk) [reply src/index.html:22]: What I should do? I'm confused. Just change <p> to <h2>, or use another tag.

Comment (nazarmatsevych) [reply src/index.html:27]: in this case you should use `<p>` tag instead of `div`

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+  line-height: 18px;
+  font-weight: bold;
+  color: var(--text-color);
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
```
Comment (nazarmatsevych) [inline src/style.css:98]: Now you can remove box-sizing from here

**Review by nazarmatsevych — APPROVED** [review]
Good work,
just small suggestions

**Pull request closed without merging**